mvvm-pattern/README.md

```diff
@@ -1,5 +1,7 @@
 # MVVM-Pattern
 
+[Zur Übersicht](../README.md)
+
 <h2>Was ist MVVM?</h2>
```

Comment:
In Markdown wird eine Überschrift der 2. Ebene anders eingeleitet:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers